code/repo_sync

```diff
@@ -294,10 +295,7 @@ def curl(url, destinationpath, onlyifnewer=False, etag=None, resume=False):
         print >> fileobj, 'fail'           # throw error if download fails
         print >> fileobj, 'dump-header -'  # dump headers to stdout
         print >> fileobj, 'speed-time = 30' # give up if too slow d/l
-        # next line does not work on windows
-        #print >> fileobj, 'output = "%s"' % tempdownloadpath
-
-        print >> fileobj, 'ciphers = HIGH,!ADH' # use only >=128 bit SSL
+        print >> fileobj, 'tlsv1'          # use only TLS 1.x
```

Comment:
I like the adding of tlsv1 conceptually -- just worried it will break some combos of OS version/curl and there's no easy way for the admin to override.